spec/Domains.tex

```diff
@@ -385,27 +375,7 @@ \subsubsection{Associative Domain Values}
 \index{domains!associative!default values}
 
 If unspecified the default value of an associative domain type is the empty 
-index set, except for enumerated domains.
-The default value of an enumerated domain type is the set of all constants
-of the corresponding enumerated type.
-
-\begin{rationale}
-The decision to have enumerated domains start fully populated was
-based on the observation that enumerations have a finite, typically
-small number of values and that it would be common to declare arrays
-with elements corresponding to each identifier in the enumeration.
-Further, in terms of usability it is simpler to clear a fully-populated
-domain than to fully populate an empty one.
-
-In addition, fully-populated constant enumerated
-domains are an important case for compiler optimizations, particularly
-if the numeric values of the enumeration are consecutive.
-\end{rationale}
-
-\begin{future}
-We may generally support a \chpl{startPopulated} parameter on associative
-domains, to unify this capability with other values.
-\end{future}
+index set.
```

Comment:
I find this sentence (If unspecified the default value...) really hard to read. Maybe it just needs a comma. Could you make it more readable somehow, while you're here?